eldritch/encounters.py

```diff
@@ -262,6 +262,111 @@ def Square7(char):
   fail = events.Nothing() # TODO: RUN!!! It's a GATE!
   return events.PassFail(char, check, success, fail)
 
+def Docks1(char):
+  check = events.Check(char, "luck", -1)
+  spell = events.Draw(char, "spells", 1)
+  return events.PassFail(char, check, spell, events.Nothing())
+def Docks2(char):
+  # TODO: you should just be able to draw two items
+  item1 = events.Draw(char, "common", 1)
+  item2 = events.Draw(char, "common", 1)
+  items = events.Sequence([item1, item2], char)
+  check = events.Check(char, "luck", -1)
+  success = events.Nothing()
+  fail = events.Arrested(char)
+  passfail = events.PassFail(char, check, success, fail)
+  return events.Sequence([items, passfail], char)
+def Docks3(char):
+  check = events.Check(char, "fight", 0)
+  dollars = events.Gain(char, {"dollars": 3}) # TODO: This is really dollars * number of successes
+  move = events.ForceMovement(char, "Merchant")
+  stamina = events.Loss(char, {"stamina": 1})
+  cond= events.Conditional(char, check, "successes", {0: events.Sequence([stamina, move], char), 1: dollars})
+  return events.Sequence([check, cond], char)
+def Docks4(char):
+  check = events.Check(char, "will", -1)
+  item = events.Draw(char, "unique", 1)
+  success = events.Sequence([events.Loss(char, {"sanity": 1}), item], char)
+  fail = events.Sequence([events.Loss(char, {"sanity": 2}), item], char)
+  return events.PassFail(char, check, success, fail)
+def Docks5(char):
+  check = events.Check(char, "speed", -1)
+  loss = events.Loss(char, {"sanity": 1})
+  return events.PassFail(char, check, events.Nothing(), loss)
+def Docks6(char):
+  check = events.Check(char, "will", 1)
+  lost = events.Nothing() # TODO: Oh No, what time is it?  Where am I?  I'm lost in Time AND Space!
+  return events.PassFail(char, check, events.Nothing(), lost)
+def Docks7(char):
+  check = events.Check(char, "luck", -1)
+  draw = events.Draw(char, "common", 1)
+  struggle = events.Loss(char, {"stamina": 3, "sanity": 1})
+  return events.PassFail(char, check, draw, struggle)
+
+def Unnamable1(char):
+  loss = events.Loss(char, {"sanity": 2})
+  # TODO: add prerequisite of ally being in deck
+  ally = events.DrawSpecific(char, "allies", "Brave Guy")
+  listen = events.Sequence([loss, ally], char)
+  return events.BinaryChoice(char, "Listen to the tale?", "Yes", "No", listen, events.Nothing())
+def Unnamable2(char):
+  check = events.Check(char, "lore", -1)
+  spell = events.Draw(char, "spells", 1)
+  clues = events.Gain(char, {"clues": 2})
+  delayed = events.LoseTurn(char)
```

Comment:
This one should be Delayed, not LoseTurn.

Reply:
not sure why I didn't do that as delayed, fixed.